tuf-spec.md

```diff
@@ -1188,35 +1188,40 @@ VERSION_NUMBER is the version number of the snapshot metadata file listed in
 the timestamp metadata file.  In either case, the client MUST write the file to
 non-volatile storage as FILENAME.EXT.
 
-  * **3.1**. **Check against timestamp metadata.** The hashes and version
-  number of the new snapshot metadata file MUST match the hashes (if any) and
```

Comment:
I think we should keep the "*if any*", because snapshot's hashes in timestamp are still optional (see https://github.com/theupdateframework/specification/pull/90).

Reply:
Ah I didn't realize the snapshot hash in timestamp was made optional. I'll restore this language after #119 lands.

Reply:
Just rebased on top of master (post-#119) and restored the "if any" in 5.3.1.